packages/iTwinUI-react/src/core/Slider/Slider.tsx

```diff
@@ -24,9 +24,23 @@ export type TrackDisplayMode =
   | 'odd-segments'
   | 'even-segments';
 
-const getPercentageOfRectangle = (rect: DOMRect, pointer: number) => {
-  const position = getBoundedValue(pointer, rect.left, rect.right);
-  return (position - rect.left) / rect.width;
+/**
+ * Determines the orientation of the vertical slider.
+ */
+export type Orientation = 'horizontal' | 'vertical';
```

Comment:
We usually don't move such types to a separate exported type, instead just inline them in the props and use `Pick<SliderProps, 'orientation'>`

Reply:
I tried implementing what you mentioned but I'm not sure if I've done it correctly. Can you please confirm it by looking the commit  with hash `628f8b311c1d8eaeb5ef695bd1de0883a02d8d44`?